src/Widgets/ColumnContainerWidget/ColumnContainerWidgetComponent.tsx

```diff
@@ -1,33 +1,76 @@
-import { provideComponent, ContentTag } from 'scrivito'
+import { provideComponent, ContentTag, connect } from 'scrivito'
 import { ColumnContainerWidget } from './ColumnContainerWidgetClass'
+import { ColumnWidgetInstance } from '../ColumnWidget/ColumnWidgetClass'
 
 provideComponent(ColumnContainerWidget, ({ widget }) => {
   const columns = widget.get('columns')
   if (!columns.length) return null
 
-  const content = columns.map((columnWidget, index) => {
-    const colSize = columnWidget.get('colSize') || 1
-    const className = widget.get('disableResponsiveAdaption')
-      ? `col-${colSize}`
-      : `col-md-${colSize}`
-    return (
-      <div key={index} className={className}>
-        <ContentTag
-          content={columnWidget}
-          attribute="content"
-          className="h-100"
-        />
-      </div>
-    )
-  })
-
-  const classNames = ['row']
-
-  if (widget.get('alignment')) {
-    classNames.push(`align-items-${widget.get('alignment')}`)
-  } else {
-    classNames.push('align-items-start')
+  const alignment = widget.get('alignment') || 'start'
+  const isResponsive = !widget.get('disableResponsiveAdaption')
+  const isFlex = widget.get('layoutMode') === 'flex'
+
+  const classNames = [`align-items-${alignment}`]
+  const flexClassName = isResponsive ? 'd-md-flex' : 'd-flex'
+  classNames.push(isFlex ? flexClassName : 'row')
+
+  return (
+    <div className={classNames.join(' ')}>
+      {columns.map((columnWidget: ColumnWidgetInstance) => {
+        const Column = isFlex ? FlexColumn : GridColumn
+        return (
+          <Column
+            key={columnWidget.id()}
+            columnWidget={columnWidget}
+            isResponsive={isResponsive}
+            isStretch={alignment === 'stretch'}
+          />
+        )
+      })}
+    </div>
+  )
+})
+
+type ColumnProps = {
+  columnWidget: ColumnWidgetInstance
+  isResponsive: boolean
+  isStretch: boolean
+}
+
+const FlexColumn = connect(function FlexColumn({
+  columnWidget,
+  isResponsive,
+  isStretch,
+}: ColumnProps) {
+  const classNames = isResponsive ? ['my-md-0', 'm-2'] : ['mx-2']
```

Comment:
The `m-2` does a margin to every column - also to the edges. With that there is "empty space" on the left and right of the column:

<img width="1624" alt="Screenshot 2024-08-06 at 09 40 23" src="https://github.com/user-attachments/assets/bff512ea-0798-4a5b-acc8-ee3d9844e18f">

Compare this to the "grid" version, where the left and right align with the other elements:

<img width="1624" alt="Screenshot 2024-08-06 at 09 40 35" src="https://github.com/user-attachments/assets/80a2b804-a4b6-45b1-a904-208d67b94fd8">

Reply:
Yes, this is intentional. Alex and I discussed this. Since there are many ways to use grid and flex which the current widget settings don't support (yet?) the idea is to provide two sane defaults as follows:
* The **grid** display is used for traditional multi-column layout, like columns in a newspaper which are aligned left and right with gaps in between.
* The **flex** display is used for horizontal stacks, with gaps between elements and spacing around.

The two will automatically look good for their respective use cases.
If you find a use case that requires flex but no outer spacing, we need to discuss the options.

Reply:
I do have this use-case in mind:
<img width="1624" alt="Screenshot 2024-08-06 at 11 18 22" src="https://github.com/user-attachments/assets/3f0c4f39-85e4-4450-8c77-11d17a65b0ee">

I would have expected, that the "Create new ticket" button is right aligned with the card-list below it.

Reply:
> Currently flex columns are optimized/intended for using them in the white cards under the input form.

Can you describe this use-case further? Maybe provide a working copy where this is visible with the new gained functionality.